src/nibpkg/read.nim

```diff
@@ -0,0 +1,18 @@
+import ./kmers
+import ./mainLookup
+import tables
+
+type 
+    Read* = object
```

Comment:
If there's nothing else in this object, this could be simply
```nim
type
    Read* = CountTable[uint32]
```
Like a `typedef`.

Reply:
Ah, cool, good point.. I think I'd my vote would be leave it an object to futureproof the API - we might want to add more fields (the read's sequence or a CountTable of compatible positions immediately came to mind).